underfs/gsg/src/main/java/alluxio/underfs/gsg/GSGUnderFileSystem.java

```diff
@@ -301,4 +303,31 @@ private ObjectStatus getBlobStatus(Blob blob) {
     return new ObjectStatus(blob.getName(), blob.getMd5() == null? DIR_HASH : blob.getMd5(),
         blob.getSize(), time);
   }
+
+  // To avoid version conflicts
```

Comment:
is did not get this, why is this method copied over here?

Reply:
Because we change the parameter of this method in 1.8, the `Mode.getUMask` in Alluxio 1.8.1 tarball do not have parameters, but the Alluxio 1.8 branch has parameter.

Reply:
add a TODO to remove this later